index.html

```diff
@@ -6,10 +6,142 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Sazish Academy</title>
     <link rel="stylesheet" href="assets/style.css">
-    <script src="assets/script.js"></script>
 </head>
+
 <body>
-    
+    <header class="header">
+        <nav class="navbar top-nave">
+            <div class="log">
+                <a href="index.html" class="logo-link"> S.Academy </a>
```

Comment:
- [ ] Kindly add the header where there is Facebook and Twitter.
- Your design
<img width="1219" alt="Screenshot 2023-05-31 at 09 16 10" src="https://github.com/akbarsazish/Sazish-Academy/assets/81444338/c4c59a53-f316-4c10-a253-f8e9c9507be0">
- Behance Design
<img width="1237" alt="Screenshot 2023-05-31 at 09 22 13" src="https://github.com/akbarsazish/Sazish-Academy/assets/81444338/f602571d-a965-4bf8-b8c1-c6dd7214537f">

Reply:
Dear @Kossi-stack, I would like to express my gratitude for reviewing the code and providing your recommendations. I appreciate your valuable insights, and I will make sure to take them into consideration. Thank you for your time and effort in reviewing the code.